platform/ui/src/contextProviders/ViewportGridProvider.tsx

```diff
@@ -317,6 +317,12 @@ export function ViewportGridProvider({ children, service }) {
     return viewportGridState;
   }, [viewportGridState]);
 
+
+  const getActiveViewportOption = (key: string, def?: string) => {
+    const { viewports, activeViewportIndex } = viewportGridState;
+    return viewports[activeViewportIndex].viewportOptions?.[key] ?? def;
+  }
+
```

Comment:
what is `key` and what is `def`? I hate the api where we are putting the defaults in the function arguments. It is unnecessary someone can basically say instead

`getActiveViewportOptions(key) ?? def`

outside when they are using it